### ISSUE TITLE
Is charSetSize field ever used?

### ISSUE THREAD

**Issue by alexeyr** [SSW-CocoR/CocoR-CPP#3]
https://github.com/SSW-CocoR/CocoR-CPP/blob/b7e87c573026eaebf9109fd0f4bfc1431ef0a0ce/src/Scanner.frame#L245 
is a field which doesn't seem to be used in generated code. Can it be safely removed?

**Comment by mingodad** [comment]
Good catch looking through several CocoR variants I have only one for FSharp uses something like :

https://github.com/blucz/coco-r-fsharp/blob/4b7e07df08f795a23e5f1015455e5c95b51b93a7/ScannerOutput.cs#L329
https://github.com/blucz/coco-r-fsharp/blob/4b7e07df08f795a23e5f1015455e5c95b51b93a7/Tab.cs#L195
